internal/pkg/otel/samples/darwin/logs_metrics_traces.yml

```
@@ -123,7 +121,7 @@ service:
 
     metrics/hostmetrics:
       receivers: [hostmetrics/system]
-      processors: [elasticinframetrics, resourcedetection, attributes/dataset, resource/process]
+      processors: [resourcedetection, attributes/dataset, resource/process]
       exporters: [elasticsearch/ecs]
```

```suggestion
      exporters: [elasticsearch/otel]
```

The `elasticsearch/ecs` exporter can be removed too